packages/react/vite.config.ts

```diff
@@ -5,12 +5,21 @@ import pkg from "./package.json";
 // import eslintPlugin from "vite-plugin-eslint";
 
 // https://vitejs.dev/config/
-export default defineConfig({
+export default defineConfig((conf) => ({
   test: {
     environment: "jsdom",
     setupFiles: ["./vitestSetup.ts"],
   },
   plugins: [react()],
+  resolve: {
```

Comment:
Could you add a comment what this is for?

Reply:
Removed, I don't think it's necessary here

Reply:
added again, was needed for vitest